test/Microsoft.Extensions.Logging.Test/LoggerFilterTest.cs

```diff
@@ -58,6 +58,96 @@ public void ChangingConfigReloadsDefaultFilter()
         }
 
```

Comment:
Yes, made sure they all have at least 1 test. I wanted to do a programmatic way of testing all the filters with each other etc. but it was going to be complex and crazy so...